test/lifecycle_test.dart

```diff
@@ -66,18 +68,14 @@ main() {
         int secondStateUpdateCalls;
         Map initialProps;
         Map newState1;
-        Map expectedState1;
```

Comment:
These variables and statements were unused and had no side-effects, so I removed them